package-lock.json

```diff
@@ -21365,6 +21388,12 @@
             "glob": "^7.1.3"
           }
         },
+        "semver": {
```

Comment:
I think this might be a mistake as well

Reply:
I get this in my local whenever i do `npm install`, can you do a clean install and check if its coming for you?

Reply:
Please remove this one, I will also clean to make sure it's not added later.

Reply:
There is a bug in master,  opened a new PR to fix this #713